modules/schematics/src/container/schema.json

```
@@ -97,6 +97,13 @@
       "type": "string",
       "default": "State",
       "description": "Specifies the interface for the state."
+    },
+    "testDepth": {
+      "description":
+        "Specifies whether to create a unit test or an integration test.",
+      "enum": ["Unit", "Integration"],
+      "type": "string",
+      "default": "Unit"
```

The current setup should remain the default for now.

```suggestion
      "default": "integration"
```